SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKDefaultLaunchConfigGenerator.cpp

```diff
@@ -178,6 +178,44 @@ bool GenerateLaunchConfig(const FString& LaunchConfigPath, const FSpatialLaunchC
 
 		Writer->WriteValue(TEXT("max_concurrent_workers"), LaunchConfigDescription.MaxConcurrentWorkers);
 
+		// Event tracing
+		if (SpatialGDKSettings->bEventTracingEnabled)
+		{
+			Writer->WriteObjectStart(TEXT("event_tracing_configuration"));
+			Writer->WriteValue(TEXT("enabled"), true);
+			if (bGenerateCloudConfig)
+			{
+				// TODO:
+			}
+			else
+			{
+				FString Path = FPaths::Combine(FPaths::ProjectSavedDir(), TEXT("EventTracing"));
+				Writer->WriteValue(TEXT("log_directory"), *Path);
+			}
+			Writer->WriteValue(TEXT("max_log_file_size_bytes"), SpatialGDKSettings->RuntimeMaxEventTracingFileSizeBytes);
+			Writer->WriteObjectStart(TEXT("event_filter_configuration"));
+
+			UEventTracingSamplingSettings* SamplingSettings =
+				SpatialGDKSettings->GetEventTracingSamplingSettings(); // TODO: Is this trouble in editor?
```

Comment:
Investigate this

Reply:
That the in-memory asset might not be getting updated, but this should be fine after some testing!